test/e2e/csv_e2e_test.go

```diff
@@ -4169,6 +4175,210 @@ var _ = Describe("ClusterServiceVersion", func() {
 
 		Expect(apiService2.Spec.CABundle).Should(Equal(apiService1.Spec.CABundle))
 	})
+
+	It("Disabling copied csvs", func() {
+		ns := &corev1.Namespace{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: genName("csv-toggle-test-"),
+			},
+		}
+
+		// Create a new operator group for the new namespace
+		operatorGroup := v1.OperatorGroup{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      genName("csv-toggle-test-"),
+				Namespace: ns.GetName(),
+			},
+		}
+
+		csv := v1alpha1.ClusterServiceVersion{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      genName("csv-toggle-test-"),
+				Namespace: ns.GetName(),
+			},
+			Spec: v1alpha1.ClusterServiceVersionSpec{
+				InstallStrategy: newNginxInstallStrategy(genName("csv-toggle-test-"), nil, nil),
+				InstallModes: []v1alpha1.InstallMode{
+					{
+						Type:      v1alpha1.InstallModeTypeAllNamespaces,
+						Supported: true,
+					},
+				},
+			},
+		}
+		Context("When an operator is installed in all namespace mode", func() {
```

Comment:
Any idea on whether we could (or should) deflate this Context to the top-level, e.g. `Describe("CSV copying is disabled")` or `Context("when CSV copying is disabled")` and update this to be a `When("an operator is installed in all namespace mode")`?

Reply:
I'm going to give this an attempt now.

Reply:
e2e has been updated to better align with Behavior-Driven Development style.